public/docs/ts/latest/guide/i18n.jade

```diff
@@ -0,0 +1,306 @@
+    but that can be done at a later stage in development or for production.
+
+  
+a(id="aot")
+.l-main-section
+:marked
+  ### Ahead of time (AoT) compilation approach
+  
+  The AoT approach relies on precompiling the templates using the `ngc` compiler and then injecting the translations into these files.
+  
+  Its main features are:
+    * Performance win: by reloading the entire app, there is no need to track and update the bindings/UI that change as a result of the locale change.
+    * The locale changes so rarely that the cost of the reload is usually incurred only once.
+    * No information/state is (typically) lost. The user is presumably changing the language because he could not understand the earlier language.
+    This means that he doesn't have unsaved information in the application.
+    * Tools support: the strings to translate are easily extracted from our templates.
+    The resulting translations are in a generic format that can be consumed both by Angular 2 and one of the many translation softwares available.
+    * Cannot support multiple languages in the same application view.
+    * We have to reload the app to change the language.
+    * Extra server side support is needed: Since we generate different precompiled files for each language, the server must perform cookie/user agent analysis
+    to decide which localized version of the application code should be returned to the user. This also causes a cache miss.
+    * The server is now responsible for determining the default localized version to serve. (e.g. cookies / geo-ip / Accept-Language header, etc.)
+  
+
+a(id="angular2-i18n")
+.l-main-section
+:marked
+  ## Angular 2 and i18n
+  
+  Whatever approach we decide to use, we must update our templates to define which strings will be translated.
+ 
+  
+a(id="i18n-directive")
+.l-main-section
+:marked
+  ### i18n directive
+  
+  The i18n directive is what tells Angular where it should inject the translations.
+  It is used by both the JiT and the AoT approaches.
+  
+  We simply add the attribute `i18n` to an element and the content string will be replaced when necessary.
+  
++makeExample('i18n/ts-snippets/i18n.config.snippets.ts', 'i18n-directive')(format=".")
+
+:marked
+  To help the translators, we can add more information about the meaning and context of this string.
+  Simply add a description to the i18n attribute:
+  
++makeExample('i18n/ts-snippets/i18n.config.snippets.ts', 'i18n-directive-desc')(format=".")
+
+:marked
+  We can add some meaning as well, separate the meaning from the description with a pipe `|`:
+  
++makeExample('i18n/ts-snippets/i18n.config.snippets.ts', 'i18n-directive-meaning')(format=".")
+
+:marked
+  Both the meaning and the description will be extracted by our messages extractor and added to the messages file.
+  It will help our translators to translate our application with a better understanding of what our text means. 
```

Comment:
"disambiguate" ?
Love this word do you think you can find a way to add it here ?